test/component/applicant/testAddress.js

```diff
@@ -21,11 +21,16 @@ describe('applicant-address', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        testHelpBlockContent.runTest('ApplicantAddress');
+        testHelpBlockContent.runTest('ApplicantAddress', true);
```

Comment:
What's that `true` for? The `runTest` method only accepts one param...

Reply:
I'll fix